src/medCore/process/medAbstractProcess.h

```diff
@@ -21,6 +21,17 @@
 
 class medAbstractProcessPrivate;
 
+namespace DATAERROR
```

Comment:
Put the enum directly in the ```medAbstractProcess``` class to use it's namespace instead of creating a separate one. That way you use ```medAbstractProcess::PIXELTYPE```.